python/tests/unit/arcticdb/version_store/test_parallel.py

```diff
@@ -312,8 +312,10 @@ def test_parallel_sortedness_checks_unsorted_data(lmdb_version_store, append, va
         with pytest.raises(SortingException):
             if append:
                 lib.append(sym, df_1, incomplete=True, validate_index=True)
+                lib.compact_incomplete(sym, True, False)
             else:
                 lib.write(sym, df_1, parallel=True, validate_index=True)
+                lib.compact_incomplete(sym, True, False)
```

Comment:
These compactions won't be necessary when the change I requested in `_store.py` is made